frame/support/procedural/src/dummy_part_checker.rs

```diff
@@ -39,11 +43,18 @@ pub fn generate_dummy_part_checker(input: TokenStream) -> TokenStream {
 		pub mod __substrate_genesis_config_check {
 			#[macro_export]
 			#[doc(hidden)]
-			macro_rules! #genesis_config_macro_ident {
+			macro_rules! #genesis_config_def_macro_ident {
 				($pallet_name:ident) => {};
 			}
+			#[macro_export]
+			#[doc(hidden)]
+			macro_rules! #genesis_config_std_macro_ident {
+				($pallet_name:ident) => {};
+			}
+			#[doc(hidden)]
+			pub use #genesis_config_def_macro_ident as is_genesis_config_defined;
 			#[doc(hidden)]
-			pub use #genesis_config_macro_ident as is_genesis_config_defined;
+			pub use #genesis_config_std_macro_ident as is_std_enabled_for_genesis;
```

Comment:
I just realize we could reexport the same macro here everywhere like this (but the macro should be renamed for better name like `no_op_macro_ident`)
```suggestion
			pub use #genesis_config_def_macro_ident as is_std_enabled_for_genesis;
```

Reply:
This would only be the case if I don't include the path, otherwise we'd probably need to modify this a bit and make it accept a token tree fragment rather than an ident.